extensions/amp-tiktok/validator-amp-tiktok.protoascii

```diff
@@ -32,7 +32,7 @@ tags: {  # <amp-tiktok>
   attr_lists: "extended-amp-global"
   attrs: {
     name: "data-src";
-    value_regex: "\\d";
+    value_regex: "(https://www\.tiktok\.com/.*)?\\d+.*";
```

Comment:
Perhaps a `^`? I know it was just a digit before but it might be nice to ensure it actually starts with either a digit or `https...`

Reply:
I thought validator regexes were automatically anchored at the beginning and end? For example, `ol@start` has the patten `[0-9]*`:

https://github.com/ampproject/amphtml/blob/dda314f0f70db489aa16b0058676c6ad1dab9395/validator/validator-main.protoascii#L1247

In the validator, if I put `_1_` as the value, it fails validation:

![image](https://user-images.githubusercontent.com/134745/126821019-4e0079c9-39d6-42da-bf7c-3ebbe02078f2.png)

So I think the `^` is already implied.

Reply:
You're correct for protoascii regexs. I was thinking of in-engine regexs. thx!